dependency/retrieval/go.mod

```diff
@@ -6,12 +6,13 @@ require (
 	github.com/Masterminds/semver v1.5.0
 	github.com/Masterminds/semver/v3 v3.2.0
 	github.com/go-enry/go-license-detector/v4 v4.3.0
+	github.com/keybase/go-crypto v0.0.0-20200123153347-de78d2cb44f4
 	github.com/onsi/gomega v1.24.1
 	github.com/package-url/packageurl-go v0.1.0
 	github.com/paketo-buildpacks/occam v0.13.3
 	github.com/paketo-buildpacks/packit/v2 v2.7.0
 	github.com/sclevine/spec v1.4.0
-	golang.org/x/crypto v0.3.0
+	golang.org/x/crypto v0.7.0
```

Comment:
Do we still need this package

Reply:
Seems like its still getting used in the tests?

Reply:
I'll take a look and see if I can bring them up to date.

Reply:
I think `go mod tidy` automatically prunes stale entries from `go.mod` (and hence `go.sum`), so we can rely on its output as a source of truth.

Reply:
Right. I just wonder if we should transition the tests to use the same library as the verifier code

Reply:
I have transitioned the tests over (took a little doing). I think everything should be square now.

Reply:
Sorry for the noise. I missed the point of the question 😳

Reply:
Thank you for doing that Forest!!